src/azure-cli-core/azure/cli/core/auth/identity.py

```diff
@@ -38,6 +38,10 @@
     "Select the account you want to log in with. "
     "For more information on login with Azure CLI, see https://go.microsoft.com/fwlink/?linkid=2271136")
 
+PASSWORD_CERTIFICATE_WARNING = (
+    "Using --password to pass service principal certificate is deprecated and will be removed in a "
+    "future release. Use --certificate instead.")
```

Comment:
Please kindly rephase the warning message. @dcaro @dbradish-microsoft

Reply:
```suggestion
PASSWORD_CERTIFICATE_WARNING = (
    "Passing the service principal certificate with `--password` is deprecated and will be removed in a future release. Please use `--certificate` instead.")
```

Reply:
Since the next breaking change release is very close, shall we give customers more time to see this warning message?

Reply:
Yes, let's give sufficient time for customers to notice the warning message